model/shared/src/main/scala/hmda/model/institution/InstitutionType.scala

```diff
@@ -7,13 +7,13 @@ import enumeratum.{ Enum, EnumEntry }
  */
 sealed trait DepositoryType extends EnumEntry
 
-// FIXME: This could also be modeled as `isDepository: Option[Boolean]`.  Which is cleaner?
 object DepositoryType extends Enum[DepositoryType] {
 
   val values = findValues
 
   case object Depository extends DepositoryType
   case object NonDepository extends DepositoryType
+  case object UndeterminedDepositoryType extends DepositoryType
```

Comment:
We won't have an issue deriving `respondentId`, since we'll no longer be determining that ourselves (I deleted the code [here](https://github.com/cfpb/hmda-platform/pull/765/files#diff-454df7adab09d0875ec5f470ac598c1aL23)).

The logic in the edits you mentioned doesn't assume only known types will be set, it checks to see if the type is one of several, and if not, the edit will not fail.  So, an `UndeterminedInstitutionType` will not cause any unexpected behavior in these validations.

As far as the concern for bad data, we can have a quick talk with David to discuss this further.  In short (from my understanding), it would be extremely difficult for us to resolve every single one of these undetermined types, and coding the backend to accept these seems like an easier approach without any huge consequences.